targeting_terminal.py

```diff
@@ -16,22 +16,27 @@
 class TargetingTerminal():
 
     def __init__(self, DRAWING_SCALE):
+        # Initalization Constants
         self.DRAWING_SCALE_ = DRAWING_SCALE
-
-        self.borderRectWidth_ = 2
         self.current_text_ = ""
-        self.fontSize_ = 15
-        self.maxTextWidth_ = 380
         self.terminal_rect_ = None
-        self.terminalHeight_ = 25
-        self.terminalRectX_ = 10
-        self.terminalRectY_ = 40
-        self.terminalWidth_ = 400
+        
+        # Terminal Visual Constants
+        self.terminal_RectX_ = 10
```

Comment:
This is like a mix of capCase and under_scores. Should be consistent, so
`self.terminal_Rectx_` should be `self.terminal_rect_x_